google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/GooglePublicKeysManager.java

```diff
@@ -21,11 +21,7 @@
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.JsonParser;
 import com.google.api.client.json.JsonToken;
-import com.google.api.client.util.Beta;
-import com.google.api.client.util.Clock;
-import com.google.api.client.util.Preconditions;
-import com.google.api.client.util.SecurityUtils;
-import com.google.api.client.util.StringUtils;
+import com.google.api.client.util.*;
```

Comment:
Although checkstyle didn't complain and we are lacking a CONTRIBUTING document to inform you of our style, we [don't use wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).